src/main/java/org/sagebionetworks/bridge/validators/ValidatorUtils.java

```diff
@@ -40,6 +40,10 @@
     static final String INVALID_LANG = "%s is not a valid ISO 639 alpha-2 or alpha-3 language code";
     static final String INVALID_HEX_TRIPLET = "%s is not in hex triplet format (ie #FFFFF format)";
     static final String HEX_TRIPLET_FORMAT = "^#[0-9a-fA-F]{6}$";
+    public static final String INVALID_STRING_LENGTH = "is longer than the allowed field length of %d characters";
```

Comment:
make it package instead of public, will need to move one method from TestUtils to ValidatorUtils

Reply:
Done